LibreHardwareMonitorLib/Hardware/Storage/DebugSmart.cs

```diff
@@ -4,6 +4,11 @@
 // Partial Copyright (C) Michael Möller <mmoeller@openhardwaremonitor.org> and Contributors.
 // All Rights Reserved.
 
+using LibreHardwareMonitor.Interop;
```

Comment:
No need for this.

Reply:
Debug build won't build without these but okay. Im thinking of putting whole file under if debug statement

Reply:
Debug build wont compile without this